notebooks/Download Data.ipynb

```diff
@@ -57,7 +57,7 @@
    "metadata": {},
    "source": [
     "## Downloading Data using Public HTTP API\n",
-    "Te users can download data without authentication directly using HTTP address. The dataset's public URL is `ttps://storage.googleapis.com/objectron`. You can use curl, request or any other downloader for this purpose.\n"
+    "Te users can download data without authentication directly using HTTP address. The dataset's public URL is `https://storage.googleapis.com/objectron`. You can use curl, request or any other downloader for this purpose.\n"
```

Comment:
Thank you for your contributions. While you are at it, can you fix the typo "Te" -> "The" as well?

Reply:
Sure